testsuite/models/testrun.py

```diff
@@ -0,0 +1,80 @@
+''' Test run module.
+'''
+import json
+
+from testsuite.extensions import db
+
+
+class TestRun(db.Model):
+    ''' The results of a single test run.
```

Comment:
It looks like you could maintain a normalized db with just one table containing columns for plan and snapshot. I don't necessarily object to having 3 tables but I'd like to understand your rationale.